examples/ios/objc/TodayExtension/TodayViewController.m

```diff
@@ -0,0 +1,81 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2015 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "TodayViewController.h"
+#import <NotificationCenter/NotificationCenter.h>
+
+#import "Tick.h"
+
+@interface TodayViewController () <NCWidgetProviding>
+
+@property (nonatomic, strong) UIButton *button;
+@property (nonatomic, strong) Tick *tick;
+@property (nonatomic, strong) RLMNotificationToken *notificationToken;
+
+@end
+
+@implementation TodayViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.preferredContentSize = CGSizeMake(0, 200.0);
+    [RLMRealm setDefaultRealmPath:[[[NSFileManager defaultManager] containerURLForSecurityApplicationGroupIdentifier:@"group.io.realm.examples.extension"] URLByAppendingPathComponent:@"extension.realm"].path];
+    self.tick = [Tick allObjects].firstObject;
+    if (!self.tick) {
+        [[RLMRealm defaultRealm] transactionWithBlock:^{
+            self.tick = [Tick createInDefaultRealmWithObject:@[@"", @0]];
+        }];
+    }
+    self.notificationToken = [self.tick.realm addNotificationBlock:^(NSString *notification, RLMRealm *realm) {
+        if (self.tick.count % 19 == 0) {
```

Comment:
could you add an explanatory comment here, and all other notification blocks doing something similar?